shared/Microsoft.Azure.Devices.Shared.NetStandard/Microsoft.Azure.Devices.Shared.NetStandard.csproj

```diff
@@ -62,9 +62,6 @@
     <Compile Include="..\Microsoft.Azure.Devices.Shared\TwinCollectionValue.cs" />
     <Compile Include="..\Microsoft.Azure.Devices.Shared\TwinJsonConverter.cs" />
     <Compile Include="..\Microsoft.Azure.Devices.Shared\TwinProperties.cs" />
-    <Compile Include="..\Microsoft.Azure.Devices.Shared\TwinState.cs">
```

Comment:
This is breaking API: we can't simply remove this type from Shared since we've already released this as public API.

Reply:
This is not the right place for this file, and its implementation is not correct. It was added by mistake when we created the `preview` version of the provisioning service client. I'll remove it, and apologize if any customer is using it in someway. All implementation must use the new correct version in the .../provisioning/service/src/Config